src/styles/main/_scorecard.scss

```diff
@@ -21,10 +21,354 @@ $sc-font-lg: 1.5rem;
 $sc-font-lgr: 4rem;
 
 // Energy source colors
-$gas-color: #C24E2B;
+$gas-color: #c24e2b;
 $electric-color: #0048ba;
 $steam-color: #f7c34e;
 
+// WIP CSS------------------------------------------------------------------------
```

Comment:
This file will mostly replaced by the CSS between the lines I'm working with, but leaving the existing CSS until further development occurs and I have more confidence in what I might need from the previous file.